src/components/ebay-notice/template.marko

```diff
@@ -0,0 +1,14 @@
+<${data.mainTag}
+    w-bind
+    aria-labelledby="${data.type}-status-${widget.elId()}"
+    class="${data.level}-notice ${data.level}-notice--${data.type} ${data.class}"
```

Comment:
To be consistent with the other components, the class creation/management should be moved to `index.js` in the `getTemplateData` method.

Reply:
I was thinking whether I should keep it in `index.js` when I sent a PR. Okay I will do it

Reply:
For the case of notice I would have to create 3 data classes as all three DOM elements would have different classes according to the input

Reply:
Maybe just experiment with how you might add classes. Here are two examples:

**Menu**
https://github.com/eBay/ebayui-core/blob/master/src/components/ebay-menu/index.js#L84-L96

**Listbox** (soon to be Select)
https://github.com/eBay/ebayui-core/blob/master/src/components/ebay-listbox/index.js#L48-L50

I'm sure you can come up with something to satisfy creating the classes. Also, remember that you can use arrays which make adding the class names very easy.

Reply:
Okay I will make the changes, thanks! :)